packages/base-map/src/index.tsx

```diff
@@ -102,40 +96,43 @@ const BaseMap = ({
         onMove={evt => setViewState(evt.viewState)}
         style={{
           display: "block",
-          width: "100%",
-          height: forceMaxHeight ? "100vh" : "100%"
+          height: forceMaxHeight ? "100vh" : "100%",
+          width: "100%"
         }}
         zoom={viewState.zoom}
       >
         {toggleableLayers.length > 0 && (
           // TODO: Mobile view
-          <Styled.LayerSelector id="filter-group" className="filter-group">
-            <div className="layers-list">
+          <Styled.LayerSelector className="filter-group" id="filter-group">
+            <ul className="layers-list">
               {toggleableLayers.map((layer: LayerProps, index: number) => {
                 return (
-                  <label htmlFor={layer.id} key={index}>
-                    <input
-                      onChange={() => {
-                        const updatedLayers = [...hiddenLayers];
-                        // Delete the layer id if present, add it otherwise
-                        updatedLayers.includes(layer.id)
-                          ? updatedLayers.splice(
-                              updatedLayers.indexOf(layer.id),
-                              1
-                            )
-                          : updatedLayers.push(layer.id);
+                  <li key={index}>
+                    {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                    <label>
+                      <input
+                        checked={!hiddenLayers.includes(layer.id)}
+                        id={layer.id}
+                        onChange={() => {
```

Comment:
See https://github.com/opentripplanner/otp-ui/pull/414#discussion_r902529702 -- I'm not sure what the benefit would be, since the memoized method would probably update on `hiddenLayers` updating right?

Reply:
This function is being recreated on each render __and also__ for each layer. I think we should use React's  `useCallback` hook that we use in some other packages to create the equivalent of class methods in function components.

Reply:
Oh yes that's true I did miss that... Thanks for explaining! Refactored in c4c1042d5d0cf6524adf1f5be035aae7e5e4de5c